packages/teams-js/CHANGELOG.md

```diff
@@ -0,0 +1,345 @@
+## 2.0.0-beta.3
```

Comment:
I think we had talked about changing the format of the CHANGELOG so that it would match how beachball adds comments to it. I don't think this has be done as part of this change. Are you planning to do that still?

Reply:
I did go through and add a # to all headers as part of this change, it's just hard to tell because it's also been moved so it just shows as a whole new file 😋

Reply:
I'm planning to hook up all of this stuff to the github release workflows we have once we merge this. 
Planning to change the 2.0.0-beta.3 ones to come in through changefiles. The rest I can just change manually if we want the older versions to have the same bulleted list formatting.

Reply:
Speaking of hooking up github release workflows- I found that Beachball doesn't execute the postbump hook if the change type was "None". So we'll need to include deleting the JSON changelog somewhere in release or a pipeline but I'm not sure which ticket to file that in, does anyone know?

Reply:
Also, I don't know if that's intended behavior or a bug. I haven't found any official docs on it. I just found out from testing it